documentation/KnownChanges.md

```diff
@@ -123,8 +123,14 @@ StyleCop Classic allowed multiple spaces to precede a comment placed at the end
 int x;    // comment
 ```
 
-StyleCop Analyzers does not currently make an exception to the SA1025 rule for this case, although this decision is
-still under review.
+It also allowed multiple spaces preceding a symbol, such as in the second line of the following code:
+
+```csharp
+int xyz = 1;
+int w   = 1;
```

Comment:
❔ Are we sure that the implementation of StyleCop Classic allowed this case?

Reply:
➡️ It does appear that it excluded operator symbols from CS1025:
https://github.com/StyleCop/StyleCop/blob/7da86d484c6487c5fff1ae84cbd41c03c5c212d8/Project/Src/StyleCop.CSharp.Rules/SpacingRules.cs#L1969C100-L1969C100